.github/workflows/continuous-integration.yml

```diff
@@ -59,14 +59,20 @@ jobs:
           arguments: shellcheck
         if: runner.os == 'Linux'
       - name: Report jacoco coverage
+        id: jacoco_report
         uses: gradle/gradle-build-action@v2
         env:
           ORG_GRADLE_PROJECT_epApiVersion: ${{ matrix.epVersion }}
-          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
         with:
-          arguments: coveralls
+          arguments: :code-coverage-report:clean codeCoverageReport --no-build-cache
         continue-on-error: true
         if: runner.os == 'Linux' && matrix.java == '11' && matrix.epVersion == '2.21.1' && github.repository == 'uber/NullAway'
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v3
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Are you sure we need the token here?  One of the main points of switch to Codecov is that once we connect NullAway with Codecov on the backend, we shouldn't need a token, so that way code coverage uploads work even for PRs made from forks.  The example [here](https://about.codecov.io/blog/setting-up-codecov-with-java-and-gradle/) does not use a token.  [This](https://github.com/codecov/codecov-action#usage) also says no token is required for public repos

Reply:
That's correct, I just checked and we don't need this.